hack/build-rpm-release.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# This script generates release zips and RPMs into _output/releases.
+# tito and other build dependencies are required on the host. We will
+# be running `hack/build-cross.sh` under the covers, so we transitively
+# consume all of the relevant envars.
+source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
+os::build::setup_env
+os::util::environment::setup_tmpdir_vars "build-rpm-release"
+
+if [[ "${OS_ONLY_BUILD_PLATFORMS:-}" == 'linux/amd64' ]]; then
+	# when the user is asking for only Linux binaries, we will
+	# furthermore not build cross-platform clients in tito
+	make_redistributable=0
+else
+	make_redistributable=1
+fi
+
+os::log::info 'Building Origin release RPMs with tito...'
+tito_tmp_dir="${BASETMPDIR}/tito"
```

Comment:
Outputs a bunch of other nonsense we don't want to keep around. I copy the important bits back at the end.